File: Unit2-06/java/Airplane.java
/*
* This is the standard airplane class.
*
* @author  Infinity de Guzman
* @version 1.0
* @since   2024-06-08
*/

/**
* This is the airplane class.
*/
public class Airplane {
    /** The internal variable that stores speed. */
    private int speed;

    /**
    * Constructor.
    */
    public Airplane() {
        // Default constructor
    }

    /**
    * Set an airplane speed.
    *
    * @param speed set speed
    */
    public void setSpeed(int speed) {
        this.speed = speed;
    }

    /**
    * Get speed.
    *
    * @return current speed
    */
    public int getSpeed() {
        return speed;
    }
}

File: Unit2-06/Airplane.ts
/*
* This is the standard airplane class.
*
* @author  Infinity de Guzman
* @version 1.0
* @since   2024-06-08
*/


// This is the airplane class.
export class Airplane {
    /** The internal variable that stores speed. */
    private speed: number

    // Constructor.
    public Airplane() {
    }

    // Set an airplane speed.
    public setSpeed(speed: number) {
        this.speed = speed
    }

    // Get speed.
    public getSpeed(): number {
        return this.speed
    }
}
